tests/test_data.py

```diff
@@ -221,16 +215,67 @@ async def test_query_id_filter(caplog):
 
     tx_hash = "0xbf71154020b6e96c6c5db54ab97c40db3b73cf80ddda235b5204cf6d63ef5da7"
 
-    # set up a report with an incorrect but plausible query id
-    btc_usd_query_id = "a6f013ee236804827b77696d350e9f0ac3e879328f2a3021d473a0b778ad78ac"
+    # we are monitoring a different feed
+    feed = btc_usd_median_feed
 
     confidence_threshold = 0.50
 
     # try to parse it
-    res = await parse_new_report_event(cfg, tx_hash, confidence_threshold, btc_usd_query_id)
+    res = await parse_new_report_event(cfg, tx_hash, confidence_threshold, feed)
 
     # parser should return None
     assert not res
     assert "skipping undesired NewReport event" in caplog.text
 
     cfg.endpoints.endpoints.remove(endpoint)
+
+
+@pytest.mark.asyncio
+async def test_parse_oracle_address_submission():
+
+    update_oracle_address_query_id = "0xcf0c5863be1cf3b948a9ff43290f931399765d051a60c3b23a4e098148b1f707"
+
+    class TellorOracleAddress:
+
+        query_id: str = update_oracle_address_query_id
+
+    cfg = TelliotConfig()
+    cfg.main.chain_id = 1
+
+    for endpoint in cfg.endpoints.find(chain_id=1):
+        cfg.endpoints.endpoints.remove(endpoint)
+
+    endpoint = RPCEndpoint(
+        1, "Mainnet", "Infura", "https://mainnet.infura.io/v3/db7ce830b1224efe93ae3240f7aaa764", "etherscan.io"
```

Comment:
Shouldn't these keys be added as env vars?

Reply:
yes, but pytest can't seem to find them for some reason unless i hardcode, in this case

Reply:
Probs should fix at some point #70